src/app/newSettings/Sections/Workspace/Teams/TeamsSection.tsx

```diff
@@ -15,22 +21,32 @@ import CreateTeamDialog from './components/CreateTeamDialog';
 import TeamDetails from './components/TeamDetails';
 import AddMemberDialog from './components/AddMemberDialog';
 import RenameTeamDialog from './components/RenameTeamDialog';
+import DeleteTeamDialog from './components/DeleteTeamDialog';
+import RemoveTeamMemberDialog from './components/RemoveTeamMemberDialog';
+import ChangeManagerDialog from './components/ChangeManagerDialog';
 
 const TeamsSection = ({ onClosePreferences }: { onClosePreferences: () => void }) => {
   const selectedWorkspace = useAppSelector(workspacesSelectors.getSelectedWorkspace);
   const isCurrentUserWorkspaceOwner = useAppSelector(workspacesSelectors.isWorkspaceOwner);
+  const user = useAppSelector((state) => state.user.user);
 
   const [teams, setTeams] = useState<WorkspaceTeamResponse>([]);
   const [isCreateTeamDialogOpen, setIsCreateTeamDialogOpen] = useState<boolean>(false);
   const [isRenameTeamDialogOpen, setIsRenameTeamDialogOpen] = useState<boolean>(false);
+  const [isDeleteTeamDialogOpen, setIsDeleteTeamDialogOpen] = useState<boolean>(false);
+  const [isRemoveTeamMemberDialogOpen, setIsRemoveTeamMemberDialogOpen] = useState<boolean>(false);
+  const [isChangeManagerDialogOpen, setIsChangeManagerDialogOpen] = useState<boolean>(false);
   const [newTeamName, setNewTeamName] = useState<string>('');
   const [renameTeamName, setRenameTeamName] = useState<string>('');
   const [isCreateTeamLoading, setIsCreateTeamLoading] = useState<boolean>(false);
   const [isRenameTeamLoading, setIsRenameTeamLoading] = useState<boolean>(false);
+  const [isDeleteTeamLoading, setIsDeleteTeamLoading] = useState<boolean>(false);
   const [isGetTeamsLoading, setIsGetTeamsLoading] = useState<boolean>(false);
   const [isGetTeamMembersLoading, setIsGetTeamMembersLoading] = useState<boolean>(false);
   const [isGetWorkspacesMembersLoading, setIsGetWorkspacesMembersLoading] = useState<boolean>(false);
   const [isAddMembersLoading, setIsAddMembersLoading] = useState<boolean>(false);
+  const [isRemoveTeamMemberLoading, setIsRemoveTeamMemberLoading] = useState<boolean>(false);
+  const [isChangeManagerLoading, setIsChangeManagerLoading] = useState<boolean>(false);
```

Comment:
Are all these states necessary for loading? Perhaps one state of charge is enough

Reply:
I use them in some functions and prefer to keep them separate to prevent unexpected behavior.